onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb

```diff
@@ -907,27 +915,27 @@
     "\n",
     "| Engine | Full Precision(MB) | Quantized(MB) |\n",
     "| --- | --- | --- |\n",
-    "| PyTorch 1.5 | 438 | 181 |\n",
-    "| ORT 1.4 | 438 | 112 |\n",
+    "| PyTorch 1.6 | 417.7 | 173.1 |\n",
+    "| ORT 1.4 | 417.7 | 106.5 |\n",
     "\n",
     "### Accuracy\n",
     "OnnxRuntime achieves a little bit better accuracy and F1 score, even though it has small model size.\n",
     "\n",
-    "| Metrics | PyTorch 1.5 Quantization | ORT 1.4 Quantization |\n",
-    "| --- | --- | --- |\n",
-    "| Accuracy | 0.84559 | 0.85784 |\n",
-    "| F1 score | 0.89340 | 0.90203 |\n",
-    "| Acc and F1 | 0.86950 | 0.87994 |\n",
+    "| Metrics | Full Precision | PyTorch 1.6 Quantization | ORT 1.4 Quantization |\n",
+    "| --- | --- | --- | --- |\n",
+    "| Accuracy | 0.86029 | 0.85784 | 0.85784 |\n",
+    "| F1 score | 0.90189 | 0.89931 | 0.90203 |\n",
+    "| Acc and F1 | 0.88109 | 0.87857 | 0.87994 |\n",
```

Comment:
It's rare to average Acc and F1.

Reply:
acc_and_f1 is a common used metric in huggingface/transformers